src/screens/SecretIconsScreen.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { FlatList, StyleSheet, View } from 'react-native';
+
+import { RegularText } from '../components';
+import { colors, Icon, normalize } from '../config';
+
+type TIconItem = {
+  iconName: string;
+};
+
+const iconNames = Object.keys(Icon);
+
+const IconItem = ({ iconName }: TIconItem) => {
+  const SelectedIcon = Icon[iconName];
+
+  return (
+    <View style={styles.iconField}>
+      <SelectedIcon size={normalize(35)} color={colors.secondary} />
+      <RegularText smallest>{iconName}</RegularText>
+    </View>
+  );
+};
+
+export const SecretIconsScreen = () => {
+  return (
+    <FlatList
+      data={iconNames}
```

Comment:
as `iconNames` is only used once and there is no computation or anything, you can throw `Object.keys(Icon)` in here directly.

Reply:
and we need to filter out `NamedIcon` i guess:

![image](https://github.com/smart-village-solutions/smart-village-app-app/assets/1942953/a5ed3718-4172-4f58-b6a3-7199031bef2e)

Reply:
filtered it out in the new commit [29186de](https://github.com/smart-village-solutions/smart-village-app-app/pull/914/commits/29186de714d420a15bf9284a4e17cbc553609203)